AnaProd/HH_bbtautau/baseline.py

```diff
@@ -75,16 +76,16 @@ def RecoHttCandidateSelection(df, config):
 
 
     df = df.Define("Muon_B2_muMu_1", f"""
-        Muon_B0 && ( (Muon_tightId && Muon_pfRelIso04_all < 0.15) || (Muon_highPtId && Muon_tkRelIso < 0.15) )
+        Muon_B0 && ( (Muon_tightId && Muon_pfRelIso04_all < 0.15) )
     """)
     df = df.Define("Muon_B2_muMu_2", f"""
-        Muon_B0 && ( (Muon_tightId && Muon_pfRelIso04_all < 0.3) || (Muon_highPtId && Muon_tkRelIso < 0.3) )
+        Muon_B0 && ( (Muon_tightId && Muon_pfRelIso04_all < 0.3) )
     """)
 
     df = df.Define("Electron_B2_eMu_1",f"Electron_B0 && Electron_mvaIso_WP80 ")
     #  Electron_mvaNoIso_WP80 && Electron_pfRelIso03_all < 0.3
```

Comment:
- do we store `mvaNoIso_WP80` and `mvaIso_WP80` for electron in anaTuple? If not, we should.
- for eMu I would keep OR for electron selection (and apply the final choice at the anaTuple level)

Reply:
we store them
ok I make this change